recipes-bsp/trusted-firmware-a/trusted-firmware-a.bb

```diff
@@ -0,0 +1,77 @@
+DESCRIPTION = "Trusted Firmware-A for Renesas RZ"
+
+require include/rzg2l-optee-config.inc
+inherit deploy
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302 \
+"
+
+S = "${WORKDIR}/git"
```

Comment:
Here is my finding:

S is the location in the Build directory where unpacked recipe source code resides. By default, this directory is {WORK_DIR }/${BPN }-${PV }.
Link document: https://docs.yoctoproject.org/ref-manual/variables.html#term-S

Because the Makefie is located in the ${WORKDIR}/git. So , if S is the default location, it results Makefile not found.